src/test/scala/fm/sbt/S3URLHandlerTest.scala

```diff
@@ -0,0 +1,19 @@
+package fm.sbt
+
+import org.scalatest.PrivateMethodTester
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers
+
+final class S3URLHandlerTest extends AnyFunSuite with Matchers with PrivateMethodTester {
+  test("getDNSAliases") {
+    val getDNSAliasesMethod = PrivateMethod[Seq[String]]('getDNSAliases)
+    S3URLHandler invokePrivate getDNSAliasesMethod("maven.custom") shouldBe Seq("s3-w.us-east-1.amazonaws.com.", "s3-1-w.amazonaws.com.")
```

Comment:
This is based upon legacy endpoints: https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html#s3-legacy-endpoints